packages/components/src/modal/index.js

```diff
@@ -108,7 +108,7 @@ class Modal extends Component {
 			children,
 			aria,
 			instanceId,
-			isDismissable,
+			isDismissible,
```

Comment:
This is a breaking change, do we want to support both for sometime and deprecate the `isDismissable`?

(check `@wordpress/deprecated` usage to see how we deprecate things)

Reply:
Sure, sorry for the mis-sight on my behalf, I will work on supporting both and deprecate `isDismissable`